transmogrifier/sources/ead.py

```diff
@@ -252,6 +252,48 @@ def get_optional_fields(self, xml: Tag) -> Optional[dict]:
                         )
                     )
 
+        # rights
+        for rights_element in collection_description.find_all(
+            ["accessrestrict", "userestrict"], recursive=False
+        ):
+            if rights_value := self.create_string_from_mixed_value(
+                rights_element, " ", ["head"]
+            ):
+                fields.setdefault("rights", []).append(
+                    timdex.Rights(
+                        description=rights_value,
```

Comment:
We're losing a whole lot of links here due to the `extref` tag issue, would you mind noting this particular field as one that should be looked at in that ticket?

Reply:
Done